backend/frontend/views.py

```diff
@@ -122,6 +122,26 @@ def can_view_user_board(logged_in_user):
     return False
 
 
+def can_view_team(logged_in_user):
+    request = get_current_request()
+    viewed_team_id = request.resolver_match.kwargs.get("team_id")
+
+    if logged_in_user.id == viewed_team_id or logged_in_user.is_superuser:
```

Comment:
Do we really need this comparison: `logged_in_user.id == viewed_team_id`

Or just that "user.is_superuser" and maybe is part of the team?